### PR TITLE
Package netutil

### PR DESCRIPTION
**Pull request by sp-jordi-forns**
- [x] Depends on #35

### CODE REVIEW

File: netutil/freeport.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+		panic(err)
+	}
+	if err := l.Close(); err != nil {
+		panic(err)
+	}
+
+	return l.Addr().(*net.TCPAddr)
+}
+
+// FreeUDPAddr returns an available TCP port
```
Comment (sp-daniel-pinyol) [inline netutil/freeport.go]: should be "...UDP port"

File: netutil/freeport.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
+package netutil
+
+import (
+	"net"
+)
+
+// FreeTCPAddr returns an available TCP port
+func FreeTCPAddr() *net.TCPAddr {
```
Comment (gnieto) [inline netutil/freeport.go]: Too much panic inside this `func`s. I guess that the unique way to trigger them is that OS has no more free ports, isn't it? If that's the case, I'm ok with them.

Comment (sp-jordi-forns) [reply netutil/freeport.go]: correct. i didn't participate in writing the original package, but that's the explanation i got back then.

Comment (ronnylt) [reply netutil/freeport.go]: That's it. Also, this is mainly intended to be used in testing.

If we are not confortable with panic-ing inside the function, maybe we can follow the `Must` pattern we see in the standard library. For example: https://golang.org/src/regexp/regexp.go?s=8678:8714#L227

**Pull request merged**